oic.r.crl.raml

```diff
@@ -66,7 +67,7 @@ traits:
       Creates the CRL data
     body:
       application/json:
-        schema: Crl
+        schema: Crl-Update
         example: |
           {
           "rt": ["oic.r.crl"],
```

Comment:
DELETE below is not really defined in the spec.  This is being tracked by Bug 2008 since /crl in general is not well defined.